### PR TITLE
feat: move stringify function to block-store-actions-functions

### PR DESCRIPTION
**Pull request by thomas9911**
should replace https://my.bettyblocks.com/block-store/4a3eabcb-2453-4851-b608-7df0edfda8c5

### CODE REVIEW

File: functions/stringify/1.0/function.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "description": "Stringify a value",
+  "label": "Stringify",
+  "category": "Misc",
+  "icon": {
+    "name": "TextIcon",
+    "color": "Purple"
+  },
+  "options": [{
```
Comment (emielwit) [inline functions/stringify/1.0/function.json:9]: Make both options required value and result

Comment (thomas9911) [reply functions/stringify/1.0/function.json]: added

File: functions/stringify/1.0/function.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{
+  "description": "Stringify a value",
```
Comment (emielwit) [inline functions/stringify/1.0/function.json:2]: Stringify a JSON value

Comment (thomas9911) [reply functions/stringify/1.0/function.json]: updated

**Review by emielwit — APPROVED** [review]

**Pull request merged**